<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="JKTLPLS009" name="test.ExtentWithTestNG" tests="2" failures="0" timestamp="2020-09-21T16:44:48 IST" time="0.053" errors="1">
  <testcase name="test1" time="0.000" classname="test.ExtentWithTestNG">
    <skipped/>
  </testcase> <!-- test1 -->
  <system-out/>
  <testcase name="setup" time="0.053" classname="test.ExtentWithTestNG">
    <error type="java.lang.NoSuchFieldError" message="configContext">
      <![CDATA[java.lang.NoSuchFieldError: configContext
at com.aventstack.extentreports.reporter.ExtentHtmlReporter.loadDefaultConfig(ExtentHtmlReporter.java:71)
at com.aventstack.extentreports.reporter.ExtentHtmlReporter.<init>(ExtentHtmlReporter.java:57)
at com.aventstack.extentreports.reporter.ExtentHtmlReporter.<init>(ExtentHtmlReporter.java:61)
at test.ExtentWithTestNG.setup(ExtentWithTestNG.java:22)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:64)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:364)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:318)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:327)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- setup -->
  <system-out/>
</testsuite> <!-- test.ExtentWithTestNG -->
